Sealnote/src/main/java/com/twistedplane/sealnote/data/Note.java

```diff
@@ -93,7 +93,7 @@ private void readFromParcel(Parcel inParcel) {
         mArchived = inParcel.readInt() > 0;
         mDeleted = inParcel.readInt() > 0;
         mType = Type.valueOf(inParcel.readString());
-        mNote = NoteContent.fromString(mType, inParcel.readString());
+        mNote = NoteContent.fromString(mType, note);
```

Comment:
Oh, I didn't see that in the original code had a note variable. Since that variable is used just once otherwise, just remove that dead binding, and keep the `mNote = NoteContent.fromString(mType, inParcel.readString());` line.